sdk/Lusid.Sdk/Client/Configuration.cs

```diff
@@ -118,7 +118,7 @@ public Configuration()
             };
 
             // Setting Timeout has side effects (forces ApiClient creation).
-            Timeout = 100000;
+            Timeout = 120000;
```

Comment:
this is part of the generated code and will get overridden each time the sdk is generated